src/components/scrollbar/Scrollbar.tsx

```diff
@@ -1,10 +1,15 @@
-import React, { forwardRef } from 'react';
+import React, { FC, forwardRef } from 'react';
 import { SScrollbar } from './styles/SScrollbar';
 
-export const Scrollbar = React.forwardRef((props, ref) => {
+interface IScrollbarProps{
+  children?,
+  ref?
+}
+
+export const Scrollbar: FC<IScrollbarProps> = forwardRef((children, ref) => {
```

Comment:
@steiningermona this doesn't look correct. What your are calling children is actually props.